src/opnsense/mvc/app/models/OPNsense/Auth/User.xml

```diff
@@ -30,6 +30,7 @@
                     <o0 value="/bin/csh">/bin/csh</o0>
                     <o1 value="/bin/sh">/bin/sh</o1>
                     <o2 value="/bin/tcsh">/bin/tcsh</o2>
+                    <o3 value="/usr/local/bin/bash">/usr/local/bin/bash (when installed)</o3>
```

Comment:
I’d really much rather have this implicit. It will be an unpluggable maintenance hotspot.

Reply:
Would it look something like this..?  Fumbling in the dark a bit as I'm not overly familiar with all of the intricacies of this MVC framework.

https://github.com/bensmithurst/opnsense-core/commit/0bb18c5badd39b05eed54c8e8edd6bbe7fad2d52